streaming/src/java/org/apache/hive/streaming/HiveStreamingConnection.java

```diff
@@ -386,12 +433,12 @@ public PartitionInfo createPartitionIfNotExists(final List<String> partitionValu
     return new PartitionInfo(partName, partLocation, exists);
   }
 
-  IMetaStoreClient getMSC() {
+  public IMetaStoreClient getMSC() {
```

Comment:
is this required to be public? exposing the metastore client via streaming connection API seems to be risky as clients can mess with it.